apps/cookbook/src/app/examples/list/components/base-list.component.ts

```diff
@@ -719,7 +719,7 @@ export class BaseListComponent {
     },
   ];
 
-  public items: any[] = this.itemsFullList.slice(0, 8);
+  public items: any[] = this.itemsFullList.slice(0, 20);
```

Comment:
![image](https://user-images.githubusercontent.com/9210691/78666620-e3995500-78d7-11ea-9959-6812bfdd9f08.png)

So the examples are engaging virtual scrolling.